bitboard/bits.go

```diff
@@ -41,12 +46,7 @@ func Flood(c *Constants, within uint64, seed uint64) uint64 {
 }
 
 func Grow(c *Constants, within uint64, seed uint64) uint64 {
-	next := seed
-	next |= (seed << 1) &^ c.R
```

Comment:
The important part here is that, since we're using `c.Mask` below, we don't also need to mask using `c.R`